pyproject.toml

```
@@ -76,7 +76,7 @@ doc = [
     "imageio>=2.30.0,<2.35",
     #"imageio-ffmpeg>=0.4.0,<0.5",
     "ipython>=7.34.0; python_version == '3.7'",
-    "ipython>=8.13.0,<8.26; python_version > '3.7'",
+    "ipython>=8.26.0,<8.27; python_version > '3.7'",
```

```suggestion
    "ipython>=8.13.0,<8.27; python_version > '3.7'",
```
We should keep the lower bound as before to avoid too tighted version ranges.